client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -22,12 +23,13 @@ const commentActions = {
 const hasAction = ( status, action ) => includes( commentActions[ status ], action );
 
 export const CommentDetailActions = ( {
+	siteId,
+	postId,
+	commentId,
```

Comment:
I typically prefer this ordering to alphabetical when it's so clearly established in an existing hierarchy. That's my vote at least.